best-practices.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
```

Comment:
I think it would be nice to move these rule categories to a sub-directory, like `/rules`